caffe2/ideep/operators/conv_transpose_op.cc

```diff
@@ -121,60 +109,36 @@ class IDEEPConvTransposeGradientOp final : public IDEEPConvTransposeUnpoolBase {
     const auto& filter = Input(FILTER);
     const auto& dY = Input(OUTPUT_GRAD);
     auto* dfilter = Output(FILTER_GRAD);
-
-    itensor dfilter_;
-    itensor filter_;
-    auto filter_in = filter;
-
-    itensor::dims oihw_dims {filter.get_dim(1), filter.get_dim(0),
-        filter.get_dim(2), filter.get_dim(3)};
-    const bool pre_converted = (filter.get_public_format() == ideep::format::iohw);
-    if (!pre_converted) {
-      auto expected_descriptor =
-            ideep::convolution_transpose_forward::expected_weights_descriptor(
-              oihw_dims,
-              filter.get_data_type(),
-              stride_,
-              pad_tl(),
-              pad_br());
-      // Framework has filters in IOHW while MKL-DNN requires OIHW,
-      // we have to do explicit conversion here.
```

Comment:
We are removing these conversions since DNNL 1.x can support IOHW weight now?

Reply:
This whole workaround regarding deconv is simply due to the lack of IOHW format tag definition before, and now they have it. Nothing has changed in the deconv primitive though.